commands/bench/bench.cmd.json

```diff
@@ -80,7 +80,7 @@
         "repos": ["polkadot-sdk"],
         "args": {
           "subcommand":   { "label": "Subcommand", "type_one_of": ["pallet"] },
-          "runtime":      { "label": "Runtime", "type_one_of": ["glutton-kusama", "glutton-kusama-dev-1300"] },
+          "runtime":      { "label": "Runtime", "type_one_of": ["glutton-kusama", "glutton-kusama-dev-1300", "glutton-westend", "glutton-westend-dev-1300"] },
```

Comment:
@seadanda kusama & polkadot could be removed?

Reply:
That change is in #52, but the native parachain runtimes are so far not removed from the repo (currently blocked due to their use in `polkadot-parachain-bin`).

We could of course just merge #52 and stop supporting benchmarking those parachain runtimes straight away - what do you think?

Reply:
yea, unless people still using them
@ggwpez wdyt?

Reply:
Lets do it in #52